### PR TITLE
profiling: add edge case resolution in known issues

### PR DESCRIPTION
**Pull request by inge4pres**
## Reason for this PR
Address the non-documented case of 8.9.0 setup (faulty) trying to upgrade to a more recent version.

### CODE REVIEW

File: docs/en/observability/profiling-get-started.asciidoc [hunk 1]
```diff
@@ -87,13 +87,22 @@ We still recommend upgrading as the latest version contains several improvements
...
-
 If your {ecloud} deployment originated from version 7.x or earlier and is currently running on version 8.9.0–8.9.2, you may encounter a problem while trying to enable Universal Profiling.
 
 Specifically, clicking the *Set up Universal Profiling* button triggers an error message that reads `Error while setting up Universal Profiling`.
+Upgrade to version 8.10.0 or more recent to fix this issue.
+
+If you previously enabled Universal Profiling in {ecloud} version 8.9.0, and you're re-executing the setup, you may face
+the same error, but upgrading to the most recent version won't be enough to fix the issue.
+
+To address the error, go to *Dev Tools* from the navigation menu and execute the following command.
```
Comment (mdbirnstiehl) [inline docs/en/observability/profiling-get-started.asciidoc:98]:
```suggestion
If you previously enabled Universal Profiling in {ecloud} version 8.9.0, and you're re-executing the setup, you may also encounter
this error. In this scenario, you need to complete the following step in addition to upgrading to the most recent version.

Open *Dev Tools* from the navigation menu and execute the following command:

```

**Review by mdbirnstiehl — APPROVED** [review]
Some wording suggestions, but otherwise LGTM.

**Comment by inge4pres** [comment]
> Some wording suggestions, but otherwise LGTM.

Thanks 🙏🏼 
Do we need to worry about the CI failing job?

File: docs/en/observability/profiling-upgrade.asciidoc [hunk 2]
```diff
@@ -77,14 +77,14 @@ Verify that no ingestion is happening by reloading the *Data Streams* and *Indic
 [[profiling-from-scratch]]
 === Set up Universal Profiling from scratch
 
-Refer to <<profiling-get-started>> to set up Universal Profiling from scratch.
+Refer to <<profiling-get-started#profiling-configure-data-ingestion,Configure Data Ingestion>> to set up Universal Profiling from scratch.
```
Comment (mdbirnstiehl) [inline docs/en/observability/profiling-upgrade.asciidoc:80]: Sorry, I missed the failing test. This should fix it.
```suggestion
Refer to <<profiling-configure-data-ingestion>> to set up Universal Profiling from scratch.

```

**Pull request merged**